devito/symbolics/inspection.py

```diff
@@ -113,7 +112,10 @@ def estimate_cost(exprs, estimate=False):
                     if all(i.function.is_const for i in terminals):
                         flops += 1
                     else:
-                        flops += trascendentals_cost.get(op.__class__, 1)
+                        try:
```

Comment:
no, I wouldn't change this line, but I'd rather change this one above:

```
trascendentals_cost = {sin: 100, cos: 100, exp: 100, log: 100}
```

something along the lines of:

```
trascendentals_cost = {i: 100 for i in <all objects in elementary.py>}
```

Reply:
I still need the pure sympy ones as well since the compiler lowers it before ending up here. (`diff2sympy`).